example/disasm/full.py

```diff
@@ -155,13 +158,13 @@
 
 
 # Generate dotty graph
-all_blocks = AsmCFG()
+asmcfg_full = AsmCFG(mdis.symbol_pool)
 for blocks in all_funcs_blocks.values():
```

Comment:
`all_funcs_blocks` is actually a list of CFG. The operation `+=` is in fact merging all of this CFG. Maybe the name could reflect that (like `all_asmcfg`)

Reply:
done